mix.exs

```diff
@@ -8,7 +8,7 @@ defmodule LoggerJSON.Mixfile do
     [
       app: :logger_json,
       version: @version,
-      elixir: "~> 1.16",
+      elixir: "~> 1.15.1",
```

Comment:
This will make it work only on `1.15.X` so what we really want is `~> 1.16 or 1.15.1`. I will fix it up after the merge so no worries about that.